### PR TITLE
Make old tracing runtimes (<700) compile

### PR DESCRIPTION
**Pull request by librelois**

### CODE REVIEW

File: tracing/300/runtime/moonbase/Cargo.toml [hunk 1]
```diff
@@ -77,7 +77,7 @@ crowdloan-rewards-precompiles = { git = "https://github.com/purestake/moonbeam",
 
 moonbeam-evm-tracer = { path = "../../../shared/runtime/evm_tracer", default-features = false, optional = true}
 evm-tracing-events = { path = "../../../shared/primitives/rpc/evm-tracing-events", default-features = false, optional = true }
```
Comment (nanocryk) [inline tracing/300/runtime/moonbase/Cargo.toml:79]: no `features = ["before_1200"]` like the other runtimes ?

File: tracing/300/runtime/moonriver/Cargo.toml [hunk 2]
```diff
@@ -75,7 +75,7 @@ crowdloan-rewards-precompiles = { git = "https://github.com/purestake/moonbeam",
 
 moonbeam-evm-tracer = { path = "../../../shared/runtime/evm_tracer", default-features = false, optional = true}
 evm-tracing-events = { path = "../../../shared/primitives/rpc/evm-tracing-events", default-features = false, optional = true }
```
Comment (nanocryk) [inline tracing/300/runtime/moonriver/Cargo.toml:77]: no `features = ["before_1200"]` like the other runtimes ?

**Review by nanocryk — CHANGES_REQUESTED** [review]

**Comment by librelois** [comment]
@nanocryk refactored to remove feature `before_1200`

**Review by nanocryk — APPROVED** [review]
Awesome ! 2 From impl are definitely better

**Comment by librelois** [comment]
@tgmichel I would like your review too

**Comment by tgmichel** [comment]
I think we should also patch `ethereum` crate in:

- tracing/700/Cargo.toml
- tracing/701/Cargo.toml
- tracing/801/Cargo.toml

`ethereum = { git = "https://github.com/purestake/ethereum", branch = "joshy-scale-info" }`

**Review by tgmichel — APPROVED** [review]
Aside from the comment lgtm!

**Comment by tgmichel** [comment]
> 

Right!

**Pull request merged**